lib/is.js

```diff
@@ -75,14 +75,24 @@ var util = require('./util');
  * Steam tests.
  */
 (function () {
-    var identifier = '[0-9\\-_]{1,100}';
+    var identifier = '[\\w-]{2,32}';
+    var numericIdentifier = '[0-9]{16}';
```

Comment:
The terminology seems to be SteamID and Steam Account Name: https://support.steampowered.com/kb_article.php?ref=1558-qyax-1965